code/game/objects/structures/fluff.dm

```diff
@@ -49,3 +49,9 @@
 	desc = "Although comfortable, this sleeper won't function as anything but a bed ever again."
 	icon = 'icons/obj/lavaland/spawners.dmi'
 	icon_state = "cryostasis_sleeper_open"
+
+/obj/structure/fluff/broken_flooring
+	name = "broken tiling"
+	desc = "A segment of broken flooring"
+	icon = 'icons/obj/brokentiling.dmi'
```

Comment:
Any direction other than south with the floor tiles gives it too much outline(plus the outline you added means it'll always have too much outline, even if it's aligned properly)